### PR TITLE
Align doc examples and fix typo in rotate

### PR DESCRIPTION
**Pull request by patrick-tolosa**
#### What does this PR solve?
Align doc examples and fix typo in rotatationMode

`import * as RotationMode from "../../../src/values/RotationMode";`
should be 
`import * as RotationMode from "../../../src/values/rotationMode";`

### CODE REVIEW

File: __DOC_EXAMPLES__/annotatedExamples/configuration.md [hunk 1]
```diff
@@ -3,9 +3,11 @@
 ```javascript
 // Import the cloudinary class, and the assets you want to use
 // In this case, we import a CloudinaryImage type.
-import {Cloudinary, CloudinaryImage} from '@cloudinary/base';
 
 // Create your instance
```
Comment (strausr) [inline __DOC_EXAMPLES__/annotatedExamples/configuration.md:7]: create your instances

Comment (strausr) [inline __DOC_EXAMPLES__/annotatedExamples/configuration.md:7]: create your instances

File: __DOC_EXAMPLES__/annotatedExamples/resize.md [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
-    cloudName: 'demo'
-  },
-  url: {
-    secure: true // force http or https
-  }
+    cloud: {
+        cloudName: 'demo'
+    },
+    url: {
+        secure: true // force http or https
```
Comment (strausr) [inline __DOC_EXAMPLES__/annotatedExamples/resize.md:14]: force https

File: __DOC_EXAMPLES__/annotatedExamples/resize.md [hunk 3]
```diff
@@ -33,7 +34,7 @@ const myImage = cld.image('sample');
 // Lets give resize() a ResizeScale paramter to scale our image
```
Comment (strausr) [inline __DOC_EXAMPLES__/annotatedExamples/resize.md:34]: should we change this from ResizeScale?

**Review by strausr — APPROVED** [review]

File: __DOC_EXAMPLES__/annotatedExamples/resize.md (hunk 3 above)
Comment (patrick-tolosa) [reply __DOC_EXAMPLES__/annotatedExamples/resize.md:34]: I'll it to the docs team to decide, but I tried to make it clear that it's a separate type of resizing,
they'll decide if to keep it or not though.

**Pull request merged**